### ISSUE TITLE
TypeScript type of call's return value doesn't match runtime type

### ISSUE THREAD

**Issue by ion1** [zksecurity/wasmati#8]
This compiles. My editor says the return type of `call(print, ...)` is `[StackVar<"i32">]`.

```ts
const print = importFunc(
  { in: [i32, i32, i32, i32, i32, i32, i32], out: [i32] },
  () => {}
);

const printCentered = func(
  { in: [i32, i32, i32, i32, i32, i32, i32], out: [i32], locals: [] },
  ([text, x, y, color, fixed, scale, small]) => {
    const [width] = call(print, [text, 0, -6, color, fixed, scale, small]);
    const newX = i32.sub(x, i32.div_s(width, 2));
    return call(print, [text, newX, y, color, fixed, scale, small]);
  }
);
```

However, it fails at runtime with `TypeError: call is not a function or its return value is not iterable`.

Changing
```diff
-    const [width] = call(print, [text, 0, -6, color, fixed, scale, small]);
+    const width: any = call(print, [text, 0, -6, color, fixed, scale, small]);
+    console.info(width);
```
makes it work and prints `{ kind: 'stack-var', id: 16, type: 'i32' }` which is indeed not an array.

**Comment by mitschabaude** [comment]
Thanks for finding that, a type bug I will fix

**Comment by mitschabaude** [comment]
Fixed and released as 0.2.3
